src/Tests/dotnet-workload-install.Tests/GivenWorkloadManifestUpdater.cs

```diff
@@ -43,48 +44,40 @@ public void GivenWorkloadManifestUpdateItCanUpdateAdvertisingManifests()
             (var manifestUpdater, var nugetDownloader, _) = GetTestUpdater();
 
             manifestUpdater.UpdateAdvertisingManifestsAsync(true).Wait();
-            var expectedDownloadedPackages = _installedManifests
-                .Select(id => ((PackageId, NuGetVersion, DirectoryPath?, PackageSourceLocation))(new PackageId($"{id}.manifest-6.0.100"), null, null, null));
-            nugetDownloader.DownloadCallParams.Should().BeEquivalentTo(expectedDownloadedPackages);
+            nugetDownloader.DownloadCallParams.Should().BeEquivalentTo(GetExpectedDownloadedPackages("6.0.100"));
         }
 
         [Fact]
         public void GivenAdvertisingManifestUpdateItUpdatesWhenNoSentinalExists()
         {
-            (var manifestUpdater, var nugetDownloader, var userProfileDir) = GetTestUpdater();
+            (var manifestUpdater, var nugetDownloader, var sentinalPath) = GetTestUpdater();
```

Comment:
Should that be sentinel instead of sentinal?

Reply:
Good question. Sentinal was used throughout the files before, but I think it technically should be sentinel... should I go back through and change it everywhere?

Reply:
Maybe a separate PR. It feels wrong to have this glaring typo in the source :)

Reply:
noticed this too: the actual sentinel file name has this same typo, so we may want to consider fixing that as well.